sklearn/preprocessing/tests/test_data.py

```diff
@@ -1243,10 +1244,20 @@ def test_binarizer():
         assert_equal(np.sum(X_bin == 0), 2)
         assert_equal(np.sum(X_bin == 1), 4)
 
+        # dtype of X is int* and binarizer will require the X to be of type
+        # float and hence inplace computation (without a copy will fail)
```

Comment:
This first test doesn't really make sense, right? was that your comment @MechCoder ?